app/config/routing.yml

```diff
@@ -1,31 +1,39 @@
 wallabag_annotation:
   type : rest
   resource: "@WallabagAnnotationBundle/Resources/config/routing_annotations.yml"
+  prefix: "%base_path%/"
 
 wallabag_import:
     resource: "@WallabagImportBundle/Controller/"
     type: annotation
-    prefix: /import
+    prefix: "%base_path%/import"
 
 wallabag_api:
     resource: "@WallabagApiBundle/Resources/config/routing.yml"
-    prefix: /
+    prefix: "%base_path%/"
 
 app:
     resource: "@WallabagCoreBundle/Controller/"
     type: annotation
+    prefix: "%base_path%/"
 
 doc-api:
     resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
-    prefix: /api/doc
+    prefix: "%base_path%/api/doc"
 
 rest :
     type : rest
     resource : "routing_rest.yml"
-    prefix : /api
+    prefix : "%base_path%/api"
+
+homepage_main:
+    path: "%base_path%/"
+    defaults:
+        _controller: WallabagCoreBundle:Entry:showUnread
+        page: 1
```

Comment:
Somehow the homepage route isn't matched when the base url is used alone with a trailing slash:

```
$ bin/console route:match /reader/
                                                                                                     
 [ERROR] None of the routes match the path "/reader/"                                                
                                                                                                     
$ bin/console route:match /reader
                                                                                                     
 [OK] Route "homepage" matches                                                                                                                                                                          

```

This seems wrong to me. My solution is a hack, but it's been a year since I worked with Symfony, so I tried the simplest thing that could work. I'd gladly accept input on finding an alternative, though.